artiq/gateware/targets/kc705.py

```diff
@@ -416,6 +416,8 @@ def main():
                         help="variant: "
                              "nist_clock/nist_qc2/sma_spi "
                              "(default: %(default)s)")
+    parser.add_argument("--identifier-str", default=None,
```

Comment:
Where is ``args.identifier_str`` used?

Reply:
Thanks for the catch. I'm running my test builds one more time.